pages/ksphere/kommander/latest/Architecture/index.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
please lowercase the directory name `Architecture` so that it is consistent with the rest of the other folders